test/oas/from-open-api.test.ts

```diff
@@ -0,0 +1,71 @@
+// @vitest-environment happy-dom
+import {
+  fromOpenApi,
+  MapOperationFunction,
+} from '../../src/open-api/from-open-api.js'
+import { createOpenApiSpec } from '../support/create-open-api-spec.js'
+import { InspectedHandler, inspectHandlers } from '../support/inspect.js'
+
+it('creates handlers based on provided filter', async () => {
+  const openApiSpec = createOpenApiSpec({
+    paths: {
+      '/numbers': {
+        get: {
+          responses: {
+            200: {
+              content: {
+                'application/json': {
+                  example: [1, 2, 3],
+                },
+              },
+            },
+          },
+        },
+        put: {
+          responses: {
+            200: {
+              content: {
+                'application/json': {
+                  example: [1, 2, 3],
+                },
+              },
+            },
+          },
+        },
+      },
+      '/orders': {
+        get: {
+          responses: {
+            200: {
+              content: {
+                'application/json': {
+                  example: [{ id: 1 }, { id: 2 }, { id: 3 }],
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  })
+
+  const mapOperation: MapOperationFunction = (url, method, operation) => {
```

Comment:
I'd put it inline into the `fromOpenApi` function. That will infer the types as well.

Reply:
89f2e80fc589ba5a59aff1b90fc3bdebd9b65203